src/Framework/Microsoft.Build.Framework.csproj

```diff
@@ -8,6 +8,7 @@
     <RootNamespace>Microsoft.Build.Framework</RootNamespace>
     <AssemblyName>Microsoft.Build.Framework</AssemblyName>
     <EnableDocumentationFile>true</EnableDocumentationFile>
+    <UpdateNuGetPackageDependencies>true</UpdateNuGetPackageDependencies>
```

Comment:
What does everyone think about this property?  It enables the NuSpec Reference Generator for a particular assembly, does the name make sense?  Should I not introduce this property and assume if a corresponding `.nuspec` exists then the functionality should be enabled?